api/transactions/transactions.go

```diff
@@ -212,11 +212,64 @@ func (t *Transactions) parseHead(head string) (thor.Bytes32, error) {
 	}
 	return h, nil
 }
+ 
+func (t *Transactions) getAllTxIDsFromTxpool() ([]string, error) {
+    txs := t.pool.Executables()
+
+    txIDs := make([]string, 0, len(txs))
+    for _, tx := range txs {
+        txIDs = append(txIDs, tx.ID().String())
+    }
+
+    return txIDs, nil
+}
+
+func (t *Transactions) handleGetAllTxIDsFromTxPool(w http.ResponseWriter, req *http.Request) error {
+    raw := req.URL.Query().Get("raw")
+    expanded := req.URL.Query().Get("expanded")
+
+    if raw != "" && raw != "false" && raw != "true" {
+        return utils.BadRequest(errors.New("raw should be boolean"))
+    }
+
+    if expanded != "" && expanded != "false" && expanded != "true" {
+        return utils.BadRequest(errors.New("expanded should be boolean"))
+    }
+
+    rawBool := raw == "true"
+    expandedBool := expanded == "true"
+
+    txs := t.pool.Executables()
```

Comment:
Use `t.pool.Dump()`. Correct me if I am wrong @qianbin.

Reply:
For pending transactions, `t.pool.Executables()` is fine.